tests/unit/store/printMyBarcode/index.spec.js

```diff
@@ -0,0 +1,25 @@
+import Store from '@/store/traction/index'
+
+describe('index', () => {
+  it('has state', () => {
+    expect(Store.state).toBeDefined()
+  })
+  it('has getters', () => {
+    expect(Store.getters).toBeDefined()
+  })
+  it('has mutations', () => {
+    expect(Store.mutations).toBeDefined()
+  })
+  it('has actions', () => {
+    expect(Store.actions).toBeDefined()
+  })
+  it('has saphyr modules', () => {
+    expect(Store.modules.saphyr).toBeDefined()
```

Comment:
Why are we testing the Store has these things that we aren't directly using?
And not printers and suffixes that we are.